run-tests.sh

```diff
@@ -6,8 +6,12 @@
 # Invenio-Records-Resources is free software; you can redistribute it and/or modify
 # it under the terms of the MIT License; see LICENSE file for more details.
 
-pydocstyle invenio_records_resources tests docs && \
-isort invenio_records_resources tests --check-only --diff && \
-check-manifest --ignore ".travis-*" && \
-sphinx-build -qnNW docs docs/_build/html && \
-pytest
+python -m pydocstyle invenio_records_resources tests docs && \
+python -m isort invenio_records_resources tests --check-only --diff && \
+python -m check_manifest --ignore ".travis-*" && \
+python -m sphinx.cmd.build -qnNW docs docs/_build/html && \
+docker-services-cli up es postgresql redis
```

Comment:
I've only needed redis when testing invenio-app-rdm (other modules didn't need it).

postgresql I would only run on travis (local tests use sqlite). 

Maybe they can be passed as arguments to not have to spin them up?

Reply:
Related to https://github.com/inveniosoftware/docker-services-cli/issues/7

Reply:
> I've only needed redis when testing invenio-app-rdm (other modules didn't need it).
Hm... could be, I'll try. It was set on the travis file and I sort of assume it was needed.

> postgresql I would only run on travis (local tests use sqlite).
As Lars mentioned it is a known issue. I'd shelve it for when we have time to go back to `docker-services-cli`.